cardano-api/src/Cardano/Api/ProtocolParameters.hs

```diff
@@ -81,12 +81,12 @@ import           Cardano.Slotting.Slot (EpochNo, EpochSize (..))
 import qualified Cardano.Ledger.Core as Ledger
 import qualified Cardano.Ledger.Era as Ledger
 import           Ouroboros.Consensus.Shelley.Eras (StandardShelley)
-import           Ouroboros.Consensus.Shelley.Protocol.Crypto (StandardCrypto)
+import           Ouroboros.Consensus.Shelley.Protocol (StandardCrypto)
```

Comment:
This one is re-exported from Cardano.Ledger.Crypto. Was unified recently. You might want to import that one instead.

Reply:
Probably it should be moved even further up into cardano-crypto or something like that I guess.

Reply:
Updated!